client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java

```diff
@@ -715,7 +716,7 @@ public void onFailure(Exception e) {
             String scrollId = searchResponse.getScrollId();
             SearchHit[] searchHits = searchResponse.getHits().getHits();
 
-            while (searchHits != null && searchHits.length > 0) { // <2>
+            while (CollectionUtils.isEmpty(searchHits) == false) { // <2>
```

Comment:
@rjernst is it an issue that this line of code is referenced in the documentation, due to the `tag::search-scroll-example`?

Reply:
Yes and no. The examples are just that: examples. We don't show the full file, so they already lack anything we include.  However, while CollectionUtils is currently brought in by the HLRC through its server jar dependency, I agree we shouldn't be leaking this method in our docs.

@zhenxianyimeng Can you please revert this and any other *DocumentationIT tests?

Reply:
@rjernst ok

Reply:
@rjernst ok